examples/scale-deployment.js

```diff
@@ -0,0 +1,29 @@
+const k8s = require('@kubernetes/client-node');
+
+const kc = new k8s.KubeConfig();
+kc.loadFromFile('./config');
```

Comment:
Can you switch this to: `kc.loadFromDefault()` since that's the happy path for most users.